### ISSUE TITLE
KCO should not prevent form submission using the enter key

### ISSUE THREAD

**Issue by forsvunnet** [krokedil/klarna-checkout-for-woocommerce#63]
This code in `assets/js/klarna-checkout-for-woocommerce.js` prevents all forms inside #kco-order-review from submitting.

	$(document).on("keypress", "#kco-order-review", function(event) {
		if (event.keyCode == 13) {
			event.preventDefault();
		}
	});

https://github.com/krokedil/klarna-checkout-for-woocommerce/blob/ea3a0188cc8dfd0a963e5c3f570df5862a5d1738/assets/js/klarna-checkout-for-woocommerce.js#L390

This causes problems when using hooks to reorganize the layout on the checkout page.

    // Eg.: Moving the coupon form to the end of the table
    remove_action( 'kco_wc_before_checkout_form', 'woocommerce_checkout_coupon_form', 20 );
    add_action( 'woocommerce_review_order_after_order_total', function() {
      ?>
      <tr>
        <td colspan="4">
          <?php woocommerce_checkout_coupon_form(); ?>
        </td>
      </tr>
      <?php
    }, 100 );

This could be fixed by targeting more specifically, eg.:

```
"#kco-order-review .qty"
```

**Comment by MichaelBengtsson** [comment]
@forsvunnet Yes that is a good suggestion. I will fix that for the next release.
